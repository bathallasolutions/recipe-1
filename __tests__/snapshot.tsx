import { render } from '@testing-library/react'
import Home from '@/pages/index'
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';

it('renders homepage unchanged', () => {
  const initialState = { counter: {value: 1} };
  const mockStore = configureStore();
  let store;

  store = mockStore(initialState);
  const { container } = render(
      <Provider store={store}>
          <Home />
      </Provider>
  );
  expect(container).toMatchSnapshot()
})
