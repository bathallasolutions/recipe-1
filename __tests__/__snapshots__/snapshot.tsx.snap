// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="container"
  >
    <main
      class="main"
    >
      <h1
        class="title"
      >
        Counter: 
        1
      </h1>
    </main>
  </div>
</div>
`;
